PocketKit/Sources/Sync/RemoteMapping/SavedItem+RemoteMapping.swift

```diff
@@ -97,7 +97,9 @@ extension SavedItem {
         }
 
         tags = NSOrderedSet(array: summary.tags?.compactMap { $0 }.map { summaryTag in
-            space.fetchOrCreateTag(byName: summaryTag.name, context: context)
+            let tag = space.fetchOrCreateTag(byName: summaryTag.name, context: context)
+            tag.update(remote: summaryTag.fragments.tagParts)
```

Comment:
all tags need remoteId

Reply:
I guess it might be better to fetch by remoteID instead of name too